src/plots/cartesian/layout_defaults.js

```diff
@@ -116,7 +116,7 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
 
     var bgColor = Color.combine(plot_bgcolor, layoutOut.paper_bgcolor);
 
-    xaList.concat(yaList).forEach(function(axName) {
+    axesList.forEach(function(axName) {
```

Comment:
Whoops! That was a missed *consideration change*. 

I changed it so that it was `xaList.concat(yaList)` but then saw that `axesList` was needed later on and goofed on the undo. I'll make a note and fix that up.